### PR TITLE
[15.0][MIG] account_bank_statement_clear_partner: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT36411

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
@pedrobaeza please review

File: account_bank_statement_clear_partner/tests/test_account_bank_statement_clear_patner.py [hunk 1]
```diff
@@ -5,11 +5,47 @@
 
 
 class TestAccountBankStatementClearPartner(common.SavepointCase):
+    @classmethod
+    def setup_multi_currency_data(cls, default_values=None, rate2016=3.0, rate2017=2.0):
+        default_values = default_values or {}
+        foreign_currency = cls.env["res.currency"].create(
```
Comment (pedrobaeza) [inline account_bank_statement_clear_partner/tests/test_account_bank_statement_clear_patner.py]: Why overcomplicating things with an extra foreign currency? Just write the same currency on both fields.

**Review by pedrobaeza — COMMENTED** [review]
Tested functionally on runbot: OK
Comment on code review.

**Comment by cesar-tecnativa** [comment]
@CarlosRoca13 please review

**Comment by cesar-tecnativa** [comment]
@LudLaf could you review this?

File: account_bank_statement_clear_partner/tests/test_account_bank_statement_clear_patner.py (hunk 1 above)
Comment (pedrobaeza) [reply account_bank_statement_clear_partner/tests/test_account_bank_statement_clear_patner.py]: This still applies. Don't create new currency. Use the same currency as the main one.

File: account_bank_statement_clear_partner/__manifest__.py [hunk 2]
```diff
@@ -9,6 +9,10 @@
     "website": "https://github.com/OCA/bank-statement-import",
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "license": "AGPL-3",
-    "depends": ["account"],
-    "data": ["views/account_bank_statement_views.xml"],
+    "depends": [
```
Comment (pedrobaeza) [inline account_bank_statement_clear_partner/__manifest__.py]: Why changing this?

**Review by pedrobaeza — COMMENTED** [review]
@LudLaf please name your commit as:

`[IMP] account_bank_statement_clear_partner: Improve README`

**Review by CarlosRoca13 — APPROVED** [review]

**Review by LudLaf — APPROVED** [review]
Functionality tested on runboat

**Comment by LudLaf** [comment]
> @LudLaf please name your commit as:
> 
> `[IMP] account_bank_statement_clear_partner: Improve README`

Commit message now renamed.

**Comment by pedrobaeza** [comment]
Too late. The PR is merged without your changes. Please create another PR with them.

**Pull request merged**